WalletWasabi.Fluent/ViewModels/Wallets/Send/TransactionPreviewViewModel.cs

```diff
@@ -97,6 +95,13 @@ public TransactionPreviewViewModel(Wallet wallet, TransactionInfo info, BitcoinA
 				}
 			});
 
+		this.WhenAnyValue(x => x.Transaction)
+			.WhereNotNull()
+			.Throttle(TimeSpan.FromMilliseconds(100))
+			.ObserveOn(RxApp.MainThreadScheduler)
+			.DoAsync(async transaction => await PrivacySuggestions.BuildPrivacySuggestionsAsync(_wallet, _info, _destination, transaction, _isFixedAmount, _cancellationTokenSource.Token))
```

Comment:
it seems similar to Subscribe, what is the difference?

Reply:
According to the Reactive documentation, we should never put any interesting code into the `Subscribe` call. If we really want to execute something then we should use `Do`.

ping @SuperJMN for more details.